themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/my-resources-page/MyResourcesPage.tsx

```diff
@@ -224,8 +227,8 @@ export class MyResourcesPage extends React.Component<MyResourcesPageProps, MyRes
         return (
             <ContentPage title="resources" onRefresh={this.fetchInitialResources.bind(this)}>
                 <Tabs isFilled activeKey={this.state.activeTabKey} onSelect={this.handleTabClick}>
-                    {this.makeTab(0, 'myResources', this.state.myResources, 'notHaveAnyResource')}
-                    {this.makeTab(1, 'sharedwithMe', this.state.sharedWithMe, 'noResourcesSharedWithYou')}
```

Comment:
Is noResourceSharedWithYou still used somewhere?  If not, it needs to be removed from the resource bundle.

Reply:
it's used on multiple places just not here ;)